css/style.css

```
@@ -3,8 +3,16 @@
 
 body {
     font-family: 'Roboto', sans-serif;
+    font-size: 16px;
```

```suggestion
    font-size: 1rem;
```
We should use relative units like `rem`, especially for `font-size`. If we use pixels, then if a user changes their browser default font size setting then the screen doesn't increase in size